README.md

```diff
@@ -21,6 +21,9 @@ The following variables influence how `ccache` is installed on the host:
   ["Configuration options" section][ccache-config-options].
 - `ccache_install_type`: Where/how to install ccache. Valid options include:
   - `build`: Build from sources
+  - `github`: Use a precompiled binary from the ccache GitHub releases page (note: not all
+    ccache versions support all platforms. Linux binaries started appearing at release
+    4.6.1, and Mac binaries at release 4.8.)
```

Comment:
Is there a patch version missing here for the Mac binaries (`4.8.?`)?

Reply:
No, it's not: https://github.com/ccache/ccache/releases/tag/v4.8 :shrug:

Reply:
@nre-ableton I think what @rco-ableton finds weird is the trailing period.
```
    4.6.1, and Mac binaries at release 4.8)
```
would probably be less confusing.

Reply:
Yes, it was the trailing dot—I assumed it was a version separator.

Reply:
Fixed in 6da0ef9.

Reply:
Ping @rco-ableton (sorry, I can't re-request a review).

Reply:
Thanks!